projects/element-ng/copyright-notice/si-copyright-notice.component.spec.ts

```diff
@@ -48,7 +48,6 @@ describe('SiCopyrightNoticeComponent', () => {
     const copyrightElement = element.querySelector<HTMLElement>('si-copyright-notice');
     expect(copyrightElement).toBeTruthy();
 
-    expect(copyrightElement?.innerText).toContain('Sample Company');
```

Comment:
CSS content does not show up in `innerText`, so we can no longer check it here

Reply:
Please keep this test and add `Sample Company` explicitely here in the provider above. We should still test this functionality in the unit test. The CSS variable replacement can be tested via VRTs, ok?

Reply:
done